main.cpp

```diff
@@ -79,42 +80,91 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 /*
  1)
  */
-
+bool playSoccer(bool soccerField, int soccerBall = 1 , int soccerPlayers = 2 )
+    {
+         ignoreUnused( soccerField , soccerBall , soccerPlayers ); 
+         return {};
+    }
 /*
  2)
- */
+ */
+ void petName(char firstLetterFirstName, char firstLetterLastName = 'H')
+ {
+     ignoreUnused( firstLetterFirstName , firstLetterLastName );
+ }
 
 /*
  3)
- */
+ */
+ int getHeight (int height , int shoeHieght = 5)
+ {
+     ignoreUnused( height , shoeHieght );
+     return {};
+ }
+
 
 /*
  4)
- */
+ */
+ int pokerWinnings (int potSize, bool didBet = false, int moneySpent = 100 )
+ {
+     ignoreUnused( potSize , didBet , moneySpent );
+     return {};
+ }
 
 /*
  5)
  */
+bool runBall (int numPlayers, bool football = true, int currentScore = 7, int otherTeamScore = 14 )
+{
+    ignoreUnused( numPlayers , football , currentScore , otherTeamScore );
+    return {};
+}
 
 /*
  6)
  */
+int gameScore (int shotsMade = 0, int shotsMissed = 0)
+{
+    ignoreUnused( shotsMade , shotsMissed );
+    return {};
+}
+
 
 /*
  7)
  */
+int getchord (int baseNote , int midNote , int topNote = 147)
+{
```

Comment:
use `camelCase` when spelling names that are the equivalent of multiple words in english.
In english, we would write `get chord`, so we must use `getChord` when spelling the function name.
- [ ] revise